accounts/abi/bind/backends/simulated.go

```diff
@@ -750,6 +750,11 @@ func (b *SimulatedBackend) SendTransaction(ctx context.Context, tx *types.Transa
 	return nil
 }
 
+// a replica of above `SendTransaction` function
+func (b *SimulatedBackend) SendTransactionConditional(ctx context.Context, tx *types.Transaction, knownAccounts map[string]map[common.Address]interface{}) error {
```

Comment:
Can you start the comment with the function name, e.g. `SendTransactionConditional is a replica...`

Reply:
[Done](https://github.com/maticnetwork/bor/pull/945/commits/abe87af0bb99b7995b621af5eb41d50b4a964e0f).